<launch>
    <param name="publishDistanceMap" type="bool" value="1"/>
    <param name="publishVoronoiMap" type="bool" value="0"/>
    
    <arg name="rviz" default="true" />

    <!-- 启动map_server节点 -->
    <node name="map_server" pkg="map_server" type="map_server" output="screen" args="$(find sentinel)/map/test.yaml">
        <remap from="map" to="map_static"/>
    </node>

    <node pkg="sentinel" type="minCostPlanner" name="minCostPlanner" output="screen" >
        <rosparam file="$(find sentinel)/param/DistanceManager.yaml" command="load" />
    </node>

    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find sentinel)/rviz/path_test.rviz" />
    </group>  
</launch>
